Pre commit fixes

EduKav1813 opened the pull request.

Commit:
mkdocs.yml: fix indentation per yamllint

Commit:
LICENSE.md: format per markdownlint-fix

Commit:
docs/: fix line length per markdownlint-fix

Commit:
docs: fix formatting per markdownlint-fix

Commit:
docs: remove missing link for Plarform Security Summit 2019 video

Commit:
.pre-commit-config.yaml: disable skip of spellcheck

Commit:
.pre-commit-config.yaml: make lychee-docker skip on ci

m-iwanicki commented on docs/events.md:
```diff
@@ -44,7 +44,7 @@ Events that TrenchBoot appeared on.
 - Platform Security Summit 2019
     + Video:
-    [Less-Insecure Network Edge Virtualization with Low Size, Weight and Power](<>)
+    Less-Insecure Network Edge Virtualization with Low Size, Weight and Power
```
Are you sure it's correct? It's better that pre-commit complains than to forget missing link

EduKav1813 replied:
Hmm, makes sense. I will reintroduce the missing link.

m-iwanicki replied:
@EduKav1813 Or maybe ask @pietrushnic. I think he was presenter at least according to https://www.platformsecuritysummit.com/2019/

EduKav1813 replied:
Fixed in 4f8f2b1dc1c247261dee3bd95197b0c9b9adadd2

m-iwanicki commented on docs/blueprints/AMD_Secure_Kernel_Loader.md:
```diff
@@ -2,97 +2,104 @@
-9. jump to the entry point of kernel
+1. initialize TPM interface
+1. initialize TPM event log
+   - log what was done by SKINIT
```
Why 3 spaces?

EduKav1813 replied:
I think it's because in both cases it wants the next row to start under the previous one. I think it's fine because it's consistent. But you can try to change that if you know how.

m-iwanicki replied:
Maybe I asked wrong, why 3 spaces instead of 4.

> I think it's fine because it's consistent

It isn't consistent with rest of the documents.

EduKav1813 replied:
Manually fixed in fd82665277fc7d1e920832ee7fb29008484e3470

The pre-commit still passes with the change

m-iwanicki commented on docs/blueprints/AMD_Grub_Late_Launch.md:
```diff
@@ -1,34 +1,34 @@
-    2. set registers as required by SKINIT
-    3. execute SKINIT as final instruction
+- Extend the late launch loader,
+  1. determine CPU type and select SKINIT or SENTER path
```
why 2 spaces?

EduKav1813 replied:
fd82665277fc7d1e920832ee7fb29008484e3470 also should cover this

m-iwanicki reviewed (changes requested):
In some documents you use 2 space tabs in others 3 space tabs.
I guess it looks a little nicer, but I think they should be the same size everywhere.

Commit:
Revert "docs: remove missing link for Plarform Security Summit 2019 video"

This reverts commit 51a3fe5686ef7f1fff497c30054e0beb7a3a706b.

Commit:
blueprints/AMD_Secure_Kernel_Loader.md: manually fix list indentation and indices

Commit:
docs/events.md: Remove the 'Video' bulletpoint

The video for Platform Security Summit 2019 was never published, so the
bulletpoint mentioning the video was removed.

EduKav1813 commented:
Removed bullet point for Video from Platform Security Summit 2019
